### PR TITLE
Support all file formats in CorruptThenRepiar nemesis

### PR DESCRIPTION
**Pull request by juliayakovlev**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by roydahan** [comment]
looks ok.
What about deleting more than one file set?

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -280,8 +280,11 @@ def _choose_file_for_destroy(self, ks_cfs):
                 continue
             file_path_splitted = file.split('/')
             # The file name like: /var/lib/scylla/data/scylla_bench/test-f60e4f30c98f11e98d46000000000002/mc-220-big-Data.db
-            # For corruption we need to remove all files that their names are started from "mc-220-"
-            file_name = '-'.join(file_path_splitted[-1].split('-')[:2]) + '-*'
+            # For corruption we need to remove all files that their names are started from "mc-220-" (MC format)
```
Comment (bentsi) [inline sdcm/nemesis.py]: need improvement here 
this is unreadable: file_path_splitted[-1][:position+2]
use specific os functions for this
```
In [6]: os.path.basename("/var/lib/scylla/data/scylla_bench/test-f60e4f30c98f11e98d46000000000002/mc-220-big-Data.db")
Out[6]: 'mc-220-big-Data.db'
```
use normal regex to find the sstable generation number:

```
In [19]: re.search("mc-(\d+)-big.*", sstable_file).group(1)
Out[19]: '220'

```

Comment (bentsi) [reply sdcm/nemesis.py]:
```
In [7]: re.search("(.*-\d+)-", "mc-220-big-Data.db").group(1)
Out[7]: 'mc-220'

```

**Comment by juliayakovlev** [comment]
@roydahan  @bentsi 
Fixed, please, review

File: sdcm/nemesis.py (hunk 1 above)
Comment (juliayakovlev) [reply sdcm/nemesis.py]: Fixed

**Comment by juliayakovlev** [comment]
> looks ok.
> What about deleting more than one file set?

Delete 5 df

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -303,13 +307,15 @@ def _destroy_data_and_restart_scylla(self):
...
         try:
-            file_for_destroy = self._choose_file_for_destroy(ks_cfs)
-
-            result = self.target_node.remoter.run('sudo rm -f %s' % file_for_destroy)
-            if result.stderr:
-                raise FilesNotCorrupted('Files were not corrupted. CorruptThenRepair nemesis can\'t be run. '
-                                        'Error: {}'.format(result))
-            self.log.debug('Files {} were destroyed'.format(file_for_destroy))
+            # Remove 5 data files
+            for _ in xrange(5):
```
Comment (bentsi) [inline sdcm/nemesis.py]: why 5 files?

Comment (juliayakovlev) [reply sdcm/nemesis.py]: Roy asked to remove 5 files, not 1. We agreed about 5

**Pull request merged**